package/upgrade/upgrade_manifests.sh

```diff
@@ -624,14 +624,11 @@ pause_all_charts() {
 
 upgrade_addons()
 {
- local is_upgrade_required=$(lower_version_check $UPGRADE_PREVIOUS_VERSION v1.1.1)
- if [ ! -z "$is_upgrade_required" ]; then
-   wait_for_addons_crd
+  wait_for_addons_crd
```

Comment:
`wait_for_addons_crd` is also only needed in specific version, but it seems harmless to wait again in later version, we may remove it in future.